spec/requests/shopify/export_product_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+require 'active_resource/base_decorator'
+
+describe 'Export a Spree Product to Shopify' do
+  include_context 'shopify_exporter_helpers'
+  include_context 'shopify_helpers'
+
+  let(:spree_product) { create(:product, name: 'Product Name') }
+  subject { find_shopify_product(spree_product) }
+
+  it 'creates a new product' do
+    export_product!(spree_product)
+    expect(subject).to be_truthy
+    cleanup_shopify_product!(subject)
```

Comment:
I tried but there are certain scenarios in the same file that are not required to be cleanup, because we clean it up in the middle of the spec thus causing spec failures.